README.md

```diff
@@ -65,24 +64,23 @@ Steps:
 
 
 ### Transfer using a third-party tool
-The Universal Transfer Operator can also offer an interface to generic third-party services that transfer data, similar to Fivetran.
+The `UniversalTransferOperator` can also offer an interface to generic third-party services that transfer data, similar to Fivetran.
 
 Here is an example of how to use Fivetran for transfers:
 
-https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_dag_fivetran.py#L52-L58
-
+https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_dag_fivetran.py#L55-L61
```

Comment:
Can we also change this URL to fetch it from a specific commit ID?

Reply:
[Done](https://github.com/astronomer/apache-airflow-provider-transfers/pull/50/commits/4b9427e8259f407873530389d2686f9b4597a10a)